src/test/regress/expected/multi_subquery_complex_queries.out

```diff
@@ -2025,8 +2024,8 @@ FROM     (
                  ) user_id 
 ORDER BY 1, 
          2 limit 10;
-ERROR:  cannot push down this subquery
-DETAIL:  Window functions are currently unsupported
+ERROR:  cannot perform distributed planning on this query
```

Comment:
Now error message does not match with the test comment. Maybe, we can add group by to the subquery to error out on window function?